doc/10-icinga-template-library.md

```diff
@@ -480,6 +487,7 @@ fping_bytes	| **Optional.** The size of ICMP packet.
 fping_target_timeout | **Optional.** The target timeout in milli-seconds.
 fping_source_ip | **Optional.** The name or ip address of the source ip.
 fping_source_interface | **Optional.** The source interface name.
+fping_extra_opts | **Optional.** Read extra plugin options from an ini file.
```

Comment:
But – of course – not in tables like this one (as they’re already kinda hopeless by themselves).